buffalo/cmd/generate/grift.go

```diff
@@ -0,0 +1,50 @@
+package generate
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/gobuffalo/buffalo/generators/grift"
+	"github.com/gobuffalo/makr"
+	"github.com/markbates/inflect"
+	"github.com/spf13/cobra"
+)
+
+//GriftCmd is the command called with the generate grift cli.
+var GriftCmd = &cobra.Command{
+	Use:     "grift [name]",
+	Aliases: []string{"grift", "g"},
+	Short:   "Generates a grift task",
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		if len(args) < 1 {
+			return errors.New("you need to provide a name for the grift tasks")
+		}
+
+		var parts []string
+		plain := strings.Contains(args[0], ":") == false
+		filename := fmt.Sprintf("%v.go", inflect.Underscore(args[0]))
+
+		if !plain {
+			parts = strings.Split(args[0], ":")
+			filename = fmt.Sprintf("%v.go", inflect.Underscore(parts[0]))
```

Comment:
file name should be the last part, not the first. so `buffalo g task db:seed` should generated `grifts/seed.go`